src/mam4xx/aero_modes.hpp

```diff
@@ -94,7 +94,7 @@ static constexpr Real mam4_coarse_mead_std_dev = 1.8;
 static constexpr Real mam4_primary_carbon_min_diameter_m = 1e-8;
 static constexpr Real mam4_primary_carbon_nom_diameter_m = 5e-8;
 static constexpr Real mam4_primary_carbon_max_diameter_m = 1e-7;
-static constexpr Real mam4_primary_carbon_mead_std_dev = 1.6;
+static constexpr Real mam4_primary_carbon_mead_std_dev = 1.60000002384186;
```

Comment:
I wonder if the last digit in these numbers is noise and if they are machine-dependent.

Reply:
It is not machine-dependent. This number is picked up from the netcdf file that reads the modal data (see `sigmag` below from the netcdf file `mam4_mode4_rrtmg_c130628.nc`):
```
sigmag = 1.60000002384186 ;

 opticsmethod = "modal                           " ;

 dgnum = 5.00000005843049e-08 ;

 dgnumlo = 9.99999993922529e-09 ;

 dgnumhi = 1.0000000116861e-07 ;

 rhcrystal = 0.35 ;

 rhdeliques = 0.8 ;
```

It may be okay to let go of the last digits, but keeping them can help with the validation testing and also finding other discrepancies.